invenio_access/__init__.py

```diff
@@ -142,6 +142,34 @@
 >>> admin_identity.can(permission_index)
 True
 
+How to discover existing actions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Modules that intend to implement a given set of actions can register them in
+entry points in the corresponding `setup.py`, e.g.
+
+.. code-block:: python
+
+    entry_points={
+        'invenio_access.actions': [
+            'records_read_all'
+            ' = invenio_records.permissions:records_read_all',
+            'records_create_all'
+            ' = invenio_records.permissions:records_create_all',
+            'records_update_all'
+            ' = invenio_records.permissions:records_update_all',
+            'records_delete_all'
+            ' = invenio_records.permissions:records_delete_all',
+        ],
+    }
+
+In order to discover which actions are available in a given installation one
+can retrieve them via:
+
+>>> app.extensions['invenio-access'].actions
+{'admin-access': Need(method='action', value='admin-access'),
```

Comment:
just be careful with printing `dict`s as the output might be different per platform/python version.